src/lib/media-import/status.ts

```diff
@@ -306,28 +308,30 @@ Downloading errors details from ${ fileErrorsUrl }
 	}
 
 	async function promptFailureDetailsDownload( fileErrorsUrl: string ) {
-		const failureDetails = await prompt( {
-			type: 'confirm',
-			name: 'download',
-			message:
-				'Download import errors report now? (Report will be downloadable for up to 7 days from the completion of the import)',
-		} );
+		if ( ! force ) {
```

Comment:
If the default value for this `false` then will the user even know that they _could_ download the report?

Reply:
If the value of `force` is false, then the user would see the prompt, which tells the user this:
`Download import errors report now? (Report will be downloadable for up to 7 days from the completion of the import)`

Reply:
I don't think I understood your question tbh